package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@easypost/api",
   "description": "EasyPost Node Client Library",
-  "version": "6.6.1",
+  "version": "6.7.0",
```

Comment:
This need to be rollback, we will have a separate to update these version when we release it